src/test/java/gov/healthit/chpl/aqa/stepDefinitions/ManageDevelopersAndProductsSteps.java

```diff
@@ -0,0 +1,55 @@
+package gov.healthit.chpl.aqa.stepDefinitions;
+
+import static org.testng.Assert.assertTrue;
+
+import org.apache.commons.lang3.StringUtils;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.support.ui.ExpectedConditions;
+import org.openqa.selenium.support.ui.WebDriverWait;
+
+import cucumber.api.java.en.Given;
+import cucumber.api.java.en.Then;
+import gov.healthit.chpl.aqa.pageObjects.ManageDevelopersAndProductsPage;
+
+/**
+ * Class ManageDevelopersAndProductsSteps definition.
+ */
+
+public class ManageDevelopersAndProductsSteps {
+    private WebDriver driver;
+    private static final int TIMEOUT = 60;
+    private String url = System.getProperty("url");
+
+    /**
+     * Constructor creates new driver.
+     */
+    public ManageDevelopersAndProductsSteps() {
+        driver = Hooks.getDriver();
+        if (StringUtils.isEmpty(url)) {
+            url = "http://localhost:3000/";
+           }
+    }
+
+    /**
+     * Loads a listing for given Database ID.
```

Comment:
Recommend adding a section here that explains what you're waiting for in the "ExpectedCondition" section. I think that'll be helpful for reviewers and those writing new Steps